src/app/hooks/useOptimizely/useServerSide/index.ts

```diff
@@ -0,0 +1,29 @@
+import { OptimizelyContext } from '@optimizely/react-sdk';
+import { useContext } from 'react';
+import { RequestContext } from '#app/contexts/RequestContext';
+import activateExperiment from '../activateExperiment';
+
+export default (experimentName: string) => {
+  const { optimizely } = useContext(OptimizelyContext);
+  const { serverSideExperiments } = useContext(RequestContext);
+
+  if (!optimizely) return null;
+
+  if (!serverSideExperiments || serverSideExperiments.length === 0) {
+    return null;
+  }
+
+  const experiment = serverSideExperiments.find(
+    ({ experimentName: serverSideExperiment }) =>
+      serverSideExperiment === experimentName,
+  );
+
+  if (!experiment) return null;
+  const { enabled, variation } = experiment;
+
+  if (!enabled || !variation || variation === 'false') return null;
+
+  if (variation) activateExperiment(optimizely, experimentName, variation);
```

Comment:
Can we make `activateExperiment` use object parameters, instead of references? 
```suggestion
  if (variation) activateExperiment({ optimizely, experimentName, variation });
```